packages/biometric-ed25519/src/index.ts

```diff
@@ -51,6 +51,11 @@ export const createKey = async (username: string): Promise<KeyPair> => {
     setBufferIfUndefined();
     return navigator.credentials.create({ publicKey })
         .then(async (res) => {
+            if (!res) {
+                alert('Passkey process was cancelled, retry to continue account setup.');
```

Comment:
The client application should be handling the thrown error rather than having this library use `alert`.

This use case makes a good argument for custom errors (e.g. `PasskeyProcessCanceled`) that could be exported and checked against in the client application.